### PR TITLE
Feature/merchant crud/story 43

### PR DESCRIPTION
**Pull request by mintona**
**Functionality:**
* Builds on functionality from story 42 to include an Activate button next to any item on the `merchant/items` page that is inactive.
* Uses hidden params in the button click for the controller update action to know which flash message to display.

**Testing:**
* RSPEC is 100% and all functionality working in devo.

**Related Issues:**
#83

### CODE REVIEW

File: app/controllers/merchant/items_controller.rb [hunk 1]
```diff
@@ -8,10 +8,34 @@ def show
 
   def update
     item = Item.find(params[:id])
-    if item.toggle!(:active?)
```
Comment (hale4029) [inline app/controllers/merchant/items_controller.rb:11]: shame to see it go

File: app/controllers/merchant/items_controller.rb [hunk 2]
```diff
@@ -8,10 +8,34 @@ def show
...
     redirect_to merchant_items_path
   end
+
+  private
+    def deactivate?
+      params[:status] == "deactivate"
+    end
+
+    def deactivate(item)
+      if item.toggle!(:active?)
```
Comment (hale4029) [inline app/controllers/merchant/items_controller.rb:27]: oh how exciting it is there

**Review by hale4029 — APPROVED** [review]
Cool beans.

**Review by Capleugh — APPROVED** [review]
Everything looks good!

**Pull request merged**